pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManagerStatelessTest.java

```diff
@@ -212,17 +212,23 @@ public void testAddRemoveInstance() {
     String instanceName = "Server_localhost_" + NUM_SERVER_INSTANCES;
     List<String> allInstances = _helixResourceManager.getAllInstances();
     assertFalse(allInstances.contains(instanceName));
+    List<String> allLiveInstances = _helixResourceManager.getAllInstances();
```

Comment:
Same for other checks
```suggestion
    List<String> allLiveInstances = _helixResourceManager.getAllLiveInstances();
```